lib/Doctrine/DBAL/Platforms/AbstractPlatform.php

```diff
@@ -1034,8 +1034,8 @@ public function getDateDiffExpression($date1, $date2)
     /**
      * Returns the SQL to add the number of given seconds to a date.
      *
-     * @param string  $date
-     * @param integer $seconds
+     * @param string         $date
+     * @param integer|string $seconds
```

Comment:
I see what you mean by this, but `string` is not valid here. Must be something like an expression object, or a field reference. /cc @deeky666